__tests__/app.test.js

```diff
@@ -146,4 +146,14 @@ describe("/api/app", () => {
         expect(body.msg).toBe("No reviews found for review_id: 10000");
       });
   });
+  test("Post a new entry into the comment", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ author: "mallionaire", body: "hi i'm mallionaire" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toHaveProperty("author");
+        expect(body.comment).toHaveProperty("body");
+      });
+  });
```

Comment:
Missing a few more tests here:

1. What happens if the request body has an extra, unneeded property added alongside author and body? This should still be a successful request!
2. What happens if the body property is missing?
3. What happens if the username property is missing?
4. What happens if the username isn't an author in our database?
5. What happens if I try to post a comment to a valid but non-existent ID (e.g. 10000)?
6. What happens if I post a comment to an invalid ID (e.g. /api/reviews/banana/comments)?